apps/src/templates/teacherNavigation/EmptyState.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import {Heading3, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+
+import styles from './teacher-navigation.module.scss';
+import dashboardStyles from '@cdo/apps/templates/teacherDashboard/teacher-dashboard.module.scss';
+
+export interface EmptyStateItem {
```

Comment:
I'm not sure `EmptyStateItem` is descriptive of what this actually contains. Item to me reflects that this is one item in a list. 

What do you think about `EmptyStateContent` or `EmptyStateInfo` or `EmptyStateProps`?

Reply:
Agreed!  Let's go with `EmptyStateContent`!